feature/contributor/src/main/java/io/github/droidkaigi/confsched2020/contributor/ui/viewmodel/ContributorsViewModel.kt

```diff
@@ -33,28 +38,50 @@ class ContributorsViewModel @Inject constructor(
     private val contributorsLoadStateLiveData: LiveData<LoadState<List<Contributor>>> = liveData {
         emitSource(
             contributorRepository.contributorContents()
+                .dropWhileIndexed { index, value ->
```

Comment:
👍

Reply:
TBH, I wish we had a mechanism set up where if cache doesn't exist, the repository would not emit :)